src/core/CMakeLists.txt

```diff
@@ -5,12 +5,14 @@ add_subdirectory(tests)
 add_subdirectory(aio)
 add_subdirectory(rpc)
 add_subdirectory(task)
+add_subdirectory(utils)
 
 add_library(dsn_runtime STATIC
         $<TARGET_OBJECTS:dsn.core>
         $<TARGET_OBJECTS:dsn.rpc>
         $<TARGET_OBJECTS:dsn.task>
         $<TARGET_OBJECTS:dsn.perf_counter>
+        $<TARGET_OBJECTS:dsn.utils>
```

Comment:
utils must be separated from dsn.runtime. If an util relies on other non-util modules, it's not util.

Reply:
But we can't make utils a seperated module, because dsn_runtime depends on it.